sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -4,6 +4,7 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
 
 - Layout - Extract content and structure (ex. words, selection marks, tables) from documents.
 - Document - Analyze key-value pairs and entities in addition to general layout from documents.
+- Read - Read page information and detected languages from documents.
```

Comment:
Can't comment on the line where the examples section is, any opinions on adding a snippet about prebuilt-read to the examples section? I was going to add it then hesitated since I'm not sure how much that prebuilt will be used over the others.

Reply:
imo we should hold off until we figure out our strategy on prebuilt samples